### ISSUE TITLE
mWeb - For USD workspace, eReceipts toggle not auto enabled

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#52206]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 5.0.49-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/runs/view/27553&group_by=cases:section_id&group_order=asc&group_id=316342
**Issue reported by:** Applause Internal Team

## Action Performed:
1. Go to https://staging.new.expensify.com/home
2. Tap profile icon - workspaces - new workspace
3. Tap profile
4. Change currency to USD
5. Navigate to workspace settings- more features
6. Enable rules and complete flow
7. Open rules in workspace settings page
8. Note for USD workspace, ereceipts toggle not auto enabled.
9. Toggle on ereceipts

## Expected Result:
In rules page, for USD workspace, ereceipts toggle must be auto enabled.

User must be able to toggle on ereceipts

## Actual Result:
In rules page, for USD workspace, ereceipts toggle not auto enabled.

User unable to toggle on ereceipts.

## Workaround:
Unknown

## Platforms:

- [ ] Android: Standalone
- [ ] Android: HybridApp
- [x] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/652a9659-7dae-417a-93f7-5f2e1ebcca2d
  
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Tony-MK** [comment]
I think this should be handled internally as `SetWorkspaceEReceiptsEnabled` is refusing to toggle the eReceiept setting from `false` to `true`.

![`SetWorkspaceEReceiptsEnabled` Response](https://github.com/user-attachments/assets/10acd0f3-cf01-4ab7-b8a3-f1e7b196a0bf)

**Comment by MitchExpensify** [comment]
Can you confirm this happened on a Collect workspace please, @IuliiaHerets ? Thank you

**Comment by MitchExpensify** [comment]
Friendly bump @IuliiaHerets, thanks

**Comment by MitchExpensify** [comment]
Necessary pre-condition for the testing steps: Log into Expensify from a non-USD location so a non-USD workspace is created

**Comment by IuliiaHerets** [comment]
@MitchExpensify Yes, it was a Control WS, option Rules available only in paid WS

**Comment by MitchExpensify** [comment]
I don't think this is a bug because its happening to Workspaces that are created n in a non-USD currency and then only when you change it to USD eReceipts are not auto-enabled. I'll confirm as a next step

**Comment by MitchExpensify** [comment]
Asking here https://expensify.slack.com/archives/C03U7DCU4/p1731971194986329

**Comment by MitchExpensify** [comment]
Confirmed we don't expect eReceipts to be enabled retroactively for workspaces changing from non-USD to USD, closing
